### PR TITLE
feat: visual motivation conversion

### LINKED ISSUE #132: Visual Motivation Conversion
Now that motivation can be consumed in visual format. I think it time to convert other motivations to become visual as well.

- [ ] Test Failures

- [ ] Startup Greeting

- [ ] Exit Salutation

### PR DESCRIPTION
**Pull request by Unthrottled**
# Motivation

Closes #132 

# Description

Converted the: Startup and Test Failure. I did not do motivation, because I just realized I forgot about it as I was making this PR. I can spawn a separate issue.

I added more reaction/motivation categories! 
- `Shocked` & `Disappointed` for test failures.
- `Celebration` & `Smug` for test passes

#### Samples

**Shocked**
Shocked

**Disappointed**
Disappointed

**Smug**
Smug

## New Feature!
When your test fails, the next time you _may_ get a  `Smug` reaction image. You only get `Smug` reactions if you where only in a negative state (eg: test or build failures) and transitioned into a positive state.

Example: 

Peek 2020-08-04 18-34

### Welcome

Peek 2020-08-04 18-38

### CODE REVIEW

File: src/main/java/zd/zero/waifu/motivator/plugin/assets/AudibleAssetDefinitionService.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package zd.zero.waifu.motivator.plugin.assets
+
+import kotlin.random.Random
+
+object AudibleAssetDefinitionService {
+
+    private val ranbo = Random(System.currentTimeMillis())
```
Comment (zd-zero) [inline src/main/java/zd/zero/waifu/motivator/plugin/assets/AudibleAssetDefinitionService.kt:7]: question: Is there a reason why it is named `ranbo`? Or is it just a typo? I do not understand what it means and did a quick search if there's a convention for random.

**Comment by zd-zero** [comment]
> You only get Smug reactions if you where only in a negative state (eg: test or build failures) and transitioned into a positive state.

I like this lol

File: src/main/java/zd/zero/waifu/motivator/plugin/assets/AudibleAssetDefinitionService.kt (hunk 1 above)
Comment (Unthrottled) [reply src/main/java/zd/zero/waifu/motivator/plugin/assets/AudibleAssetDefinitionService.kt:7]: Ranbo

:joy: 

No there is not a good reason.

Comment (zd-zero) [reply src/main/java/zd/zero/waifu/motivator/plugin/assets/AudibleAssetDefinitionService.kt:7]: Noooooooooooo 😂

Comment (Unthrottled) [reply src/main/java/zd/zero/waifu/motivator/plugin/assets/AudibleAssetDefinitionService.kt]: _I'll fix it_ 

No Fun Allowed

**Pull request merged**